.nvmrc

```diff
@@ -1 +1 @@
-16
+16.14
```

Comment:
that's not the latest of v16
perhaps we should keep it as 16? unless there's a different reason

Reply:
It was bumped because otherwise `yarn install` would not work. I bumped it to the earliest version that would not cause issue.